state_machine/mission_phase_policy_engine.py

```
@@ -74,139 +77,148 @@ class PolicyDecision:
-        
+
         # Classify severity
         severity_level = self._classify_severity(severity_score)
-        
+
         # Determine allowed actions for this phase
-        allowed_actions = phase_config.get('allowed_actions', [])
-        forbidden_actions = phase_config.get('forbidden_actions', [])
-        
+        allowed_actions = phase_config.get("allowed_actions", [])
+        forbidden_actions = phase_config.get("forbidden_actions", [])
```

Ruff (0.14.10)

Local variable `forbidden_actions` is assigned to but never used

Remove assignment to unused variable `forbidden_actions`

(F841)